components/bounties/select-network.tsx

```diff
@@ -78,14 +86,22 @@ export default function SelectNetwork({
         .then(({ rows }) => setOptions(rows.map(networkToOption)));
   }, [chain, isCurrentDefault]);
 
+  useEffect(() => {
```

Comment:
I'd say that this useEffect can be removed in favour of adding a `then` to the previous use-effect. but then again, I also think that we can remove these two useEffects by fecthing this data from the server and passing it down to this component